src/data/members.json

```
@@ -23,5 +23,10 @@
 		"displayName": "Kai",
 		"link": null,
 		"image": null
-	}
-}
+	},
+	"shibaki": {
+		"displayName": "Shibaki",
+		"link": null,
+		"image": null
+	},
+}
```

[imo]

フォーマットに何もできてない俺が悪いので、申し訳ないけど、↓みたいにすると良いです🙏
（最後に改行入れることでEOLで警告されないようにしてます）

```suggestion
	}
}

```